packages/core/contracts/data-verification-mechanism/implementation/DesignatedVotingV2.sol

```diff
@@ -41,8 +41,6 @@ contract DesignatedVotingV2 is Stakeable, MultiCaller {
         _setStakeRole(uint256(Roles.Owner));
 
         finder = FinderInterface(finderAddress);
-
-        delegateToVoter();
```

Comment:
This change is required to enable us to deploy these DesignatedVoting contracts _before_ we do the DVM2.0 update. after the fact we need to call `delegateToVoter`. This is a public function that anyone can call.

Reply:
Is it expected to put `delegateToVoter()` back in constructor after the migration to DVM2.0 so that new users can benefit from automatic sync?